signing/setup.py

```diff
@@ -72,7 +72,9 @@ def version(default):
     author='Mic Bowman, Intel Labs',
```

Comment:
Will update this, descriptions, versioning in another PR.  This wasn't required to disable legacy tests or to build python 3 deb files, so I did not update it here.